src/sidebar/components/sidebar-content.js

```diff
@@ -44,7 +55,7 @@ function SidebarContent({
   });
   const directLinkedTab = linkedAnnotation
     ? tabForAnnotation(linkedAnnotation)
-    : null;
+    : 'annotation'; // default to 'annotation' tab otherwise
```

Comment:
I think this is the biggest question mark, 

the method this is passed to is very specific about the tab name and I think thats fine, so I think we should default to one of the 3 possible options rather than null which is somewhat meaning less... unless we have a state in which we should no tabs?

```
/**
 * Set the currently-selected tab to `tabKey`.
 *
 * @param {'annotation'|'note'|'orphan'} tabKey
 */
function selectTab(tabKey) {
  return {
    type: actions.SELECT_TAB,
    tab: tabKey,
  };
}
```